controllers/configmap_controller.go

```diff
@@ -477,6 +478,28 @@ func (r *ConfigMapReconciler) EnsureServicesConfigMapExists() error {
 	return r.createServicesConfigMapOnBootup()
 }
 
+// UpdateWICDRoleBindingNamespace updates the WICD RoleBinding to point to the right ServiceAccount namespace
+func (r *ConfigMapReconciler) UpdateWICDRoleBindingNamespace(wicdResourceName string) error {
+	roleBinding, err := r.k8sclientset.RbacV1().RoleBindings(r.watchNamespace).Get(context.TODO(), wicdResourceName,
+		meta.GetOptions{})
+	if err != nil {
+		return err
+	}
+	found := false
+	for _, serviceAccount := range roleBinding.Subjects {
+		if serviceAccount.Name == wicdResourceName {
+			found = true
+			serviceAccount.Namespace = r.watchNamespace
+		}
+	}
+	if !found {
+		return errors.Errorf("WICD %s missing reference to WICD ServiceAccount", roleBinding.Kind)
+	}
+	_, err = r.k8sclientset.RbacV1().RoleBindings(r.watchNamespace).Update(context.TODO(), roleBinding,
```

Comment:
If you are doing this only with a non-default namespace, can this be patch or would it be too complicated?